src/pages/stablecoins.js

```diff
@@ -239,7 +239,7 @@ const ButtonColumn = styled.div`
   flex-direction: column;
 `
 
-const APY = styled.p`
+const Apy = styled.p`
```

Comment:
Bikeshedding but I'd prefer to just disable the rule here and use the acronym properly.